### PR TITLE
When a category is selected but it doesn't appear in the current data, "none selected" should appear

### PR DESCRIPTION
**Pull request by xavijam**
Example, selecting BRA category:

- Before (appearing "SELECT ALL"):

screen shot 2016-02-29 at 14 36 40

- Now (appearing "NONE SELECTED"):

screen shot 2016-02-29 at 14 37 16

CR: @viddo

### CODE REVIEW

File: src/widgets/category/options/options-template.tpl [hunk 1]
```diff
@@ -4,10 +4,10 @@
   <p class="CDB-Text is-semibold CDB-Size-small u-upperCase js-textInfo">
     <% if (isLocked) { %>
       <%- totalCats %> blocked <button class="CDB-Text CDB-Size-small u-upperCase u-actionTextColor CDB-Widget-link u-lSpace js-unlock">unlock</button>
-    <% } else if (areAllRejected || rejectedCats === totalCats) { %>
+    <% } else if (areAllRejected || rejectedCatsInData === totalCats || (acceptedCatsInData === 0 && acceptedCats > 0)) { %>
```
Comment (xavijam) [inline src/widgets/category/options/options-template.tpl]: We should check that there is any category accepted but there isn't any in the current data.

**Comment by viddo** [comment]
Code wise looks legit, but I have to admit I don't really understand the conditionals in the template. Could the different states be clarified somehow?

**Comment by xavijam** [comment]
Let me try! ;)

**Comment by viddo** [comment]
:+1:

**Pull request merged**